### DISCUSSION TITLE
HOw to reset postal account password?

### DISCUSSION THREAD

**Discussion by jemerci** [postalserver/postal#2315] [Help with using Postal]
Hi Support,

Postal is a great mail server, I love it , but I have got some issue, can not complete the installation.

my vps:
Linode and Digital Ocean.
OS: ubuntu 18.04
installed:
Docker
Maria DB
RabbitMQ
Postal
Caddy

I created a postal account with the command `postal make-user`, the web login says password error, I tried to reset the password, it says "If you are the application owner check the logs for more information."

To fix this, I reset ubuntu 18.04, reinstalled postal, same problem.
I switched to another vps provider, same problem. It took two whole days of tossing and turning.

I tried to google the problem, and found very few answers.
I searched hard for stackover flow, and didn't find the answer.
I searched Similar discussions, no answer.

Please tell me how to reset the password? Or, how do I access the database in order to change this password?

Is there anything I can do?
Thanks and Good Day.

Yours,
Merci

**Comment by willpower232** [comment]
You could try `postal logs web` to see what the actual problem is. It may be that it is struggling to reach your database or RabbitMQ but you won't know specifically without the logs.

**Comment by jemerci** [comment]
Hi Willpower,

Your suggestion is great.

The problem has been solved, when setting up the postal admin account for the first time, the email is changed to the domain email and it works.

However, unexpectedly, external emails, such as gmail, In my case, do not work.

Yours,
Merci

**Reply by willpower232** [reply]
do you have an error message? what is the output of `postal status`?
